src/libopensc/card-openpgp.c

```diff
@@ -584,6 +615,14 @@ pgp_init(sc_card_t *card)
 		}
 	}
 
+	if (card->reader->atr_info.hist_bytes_len >= 7 &&
+			!(memcmp(card->reader->atr_info.hist_bytes, "Yubikey", 7)) &&
+			SC_SUCCESS == pgp_yubikey_get_version(card, &yubikey_version) &&
+			yubikey_version <= 0x050403) {
```

Comment:
the compare should test a range of version number. Do you know when it started and when it is fixed?